### PR TITLE
Pm 2019 10 form width

### LINKED ISSUE #1116: styling poll dashboard: answer remove buttons break to next line
remove-buttons should be inline with the answer
Screenshot from 2019-10-23 14-57-52

### PR DESCRIPTION
**Pull request by phillimorland**
finishes fixes on #1084
fixes #1116

### CODE REVIEW

File: liqd_product/assets/scss/_form.scss [hunk 1]
```diff
@@ -99,8 +99,6 @@ div.cke_focus {
 }
 
 .form-check__label {
-    padding-left: 1.25em;
-
```
Comment (phillimorland) [inline liqd_product/assets/scss/_form.scss]: @fuzzylogic2000 I know there was a reason we don't do this but I clicked through and couldn't see anywhere it went wrong, might check again in morning

Comment (phillimorland) [reply liqd_product/assets/scss/_form.scss]: found it, yeah can't do this

Comment (fuzzylogic2000) [reply liqd_product/assets/scss/_form.scss]: Did it change how the stuff is displayed, when a chcekbox is hidden by a button? like here in the poll-dashboard:
Screenshot from 2019-10-23 14-52-11

Comment (phillimorland) [reply liqd_product/assets/scss/_form.scss]: @fuzzylogic2000 I'm not sure but it messed up the label choice for the user so I reverted the change, I'll add in the issue that its a longer fix and has always been like that, if its a priority then I can add additional class names and do some nested styles but as currently check boxes are the same everywhere I am not overly keen and only changing them in 1 place.

Comment (fuzzylogic2000) [reply liqd_product/assets/scss/_form.scss]: No, that wasn't it, that's just currently broken...

Comment (phillimorland) [reply liqd_product/assets/scss/_form.scss]: That's a problem from the old grid system, i'll fix with this pull request

**Pull request merged**